packages/components/src/summary/number.js

```diff
@@ -41,6 +42,7 @@ const SummaryNumber = ( {
 	hrefType,
 	isOpen,
 	label,
+	labelTooltipText,
```

Comment:
Shouldn't we add this prop to the `SummaryNumber` [props definition](https://github.com/woocommerce/woocommerce-admin/blob/7dcc79290ce0f17a92aac57532f9a5c460233354/packages/components/src/summary/number.js#L173)?

Reply:
Indeed, totally spaced that bit. Thanks @octaedro 👍🏻